### PR TITLE
delete prompt styling for many resources

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#731 

two parts to this: 
* add a scrollbar when the modal is longer than the page
* make it much more unlikely that the modal ever _is_ so long by fixing `Card` padding
Screen Shot 2020-06-23 at 12 54 42 PM
Screen Shot 2020-06-23 at 12 54 52 PM

### CODE REVIEW

**Comment by vincent99** [comment]
Who's going to read all this?  Can we just list the first 5 of them and then say "and [n] others"?

**Comment by mantis-toboggan-md** [comment]
Fair point; updated to hide more than 5 names with "and n more"

**Comment by westlywright** [comment]
This is a bit nit-picky but in the Ember UI when we'd run into a situation like this we would display the additional details in a tooltip. What are your thoughts @vincent99 @mantis-toboggan-md on bringing this pattern over? I'm not overly attached to the pattern, nor do I see it being a make or break for many people, but I do think some people liked having the ability to see that extra data.

**Comment by vincent99** [comment]
I like displaying several names and then "n more" because it makes it stand out better from the single delete case.  Showing me several names makes me realize it's a bulk action, and should be more effective than the Ember ui where it just says a single "n <things>" string for n>1.

But I'm never going to read through the entire list of many items in detail to see if a particular thing is in there.. I just selected them, and clicked the action, so the point is just are you sure you want to delete multiple things.

Some views (workloads) can have mixed types selected so the message is sometimes wrong when showing the type of just the first one.  There should be some way to override that with the common term (workloads), or just not display a type name if the selection has mixed types.

(And I'd like it if the altAction/skip confirm functionality from Ember existed...)

**Comment by mantis-toboggan-md** [comment]
I'm going to go with the generic label for mixed types because it doesn't seem worth it to fiddle-fart around looking for which group some arbitrary number of types belongs to, if any; you can still see where you are in the app (what resource types you're manipulating) while the modal is open, anyway. 

I also switched from 'and n more' to 'and n other(s)' because I think it sounds better in this case.

> (And I'd like it if the altAction/skip confirm functionality from Ember existed...)
@vincent99 just to confirm, you mean some kind of 'alt-click delete to skip prompt' functionality?

**Comment by vincent99** [comment]
Result looks fine, but use one (or more) translations for the whole string.. you can combine multiple plural conditions into one string, etc (e.g. https://github.com/rancher/dashboard/blob/master/assets/translations/en-us.yaml#L298-L302).

Yes in rancher/ui that's implemented as separate `action` vs `altActions` [handlers](https://github.com/rancher/ui/blob/b0a3788e78b6189477f2b77a53ee942a0687f54a/lib/shared/addon/mixins/cattle-transitioning-resource.js#L281-L282) for the call so that it can be used for other similar things. e.g. for shell/logs altAction pops them open a new window. (this doesn't _have_ to be done now if you want to make it separate)

**Comment by mantis-toboggan-md** [comment]
Is there any visual indication that alt-clicking the button will do something other than the usual function?

**Comment by vincent99** [comment]
The things that support it, like the regular delete confirm, have a "protip" that tells you about the alt version.

**Comment by mantis-toboggan-md** [comment]
Added command+click to do alt action and protip in modal informing of this option

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -100,6 +105,16 @@ cluster:
...
 
+promptRemove:
+  attemptingToRemove: You are attemping to remove the {type}
+  andOthers: |
+    {count, plural,
+    =0 {.}
+    =1 {, and one other.}
+    other {, and {count} others.}
+    }
+  protip: 'protip: Hold the Command key while clicking delete to bypass this confirmation'
```
Comment (vincent99) [inline assets/translations/en-us.yaml:116]: Almost there.. The appropriate key is utils/platform.js -> `alternateKey`/`alternateLabel`:  meta (== command) for Mac, ctrl for Lin/Win

Comment (mantis-toboggan-md) [reply assets/translations/en-us.yaml]: Done

**Pull request merged**